server/server.js

```diff
@@ -0,0 +1,49 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const path = require('path');
+const router = require('./routes/router.js');
+
+const app = express();
+const port = 3050;
+
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(express.json());
+
+app.use(express.static(path.join(__dirname, '..', 'public')));
+
+app.get('/', (_, response) => {
+  response.send(express.static(__dirname, '..', 'public', 'index.html'));
+});
+
+// Finds a /########## string, where # => any digit
+app.get(/(?:\/\d{10})(?!.)/, (request, response) => {
+  router.getSong(request.url.match(/(?:\d{10})/), (song) => {
+    console.log('The song requested was id: \n', song[0]);
+    response.send(song);
+  });
+});
+
+// Finds a /##########/song-play string, where # => any digit
+app.get(/(?:\/\d{10}\/song-play)(?!.)/, (request, response) => {
+  router.getSong(request.url.match(/(?:\d{10})/), (song) => {
+    console.log('The song to play was id: \n', song[0]);
+    response.send(song);
+  });
+});
```

Comment:
Can you elaborate on why you have a second getSong request?  It looks like both send back a song object?

Reply:
Yes. This was an unintentional repetition due to the way I initialized my routes. This has since ben corrected to make more sense and be more readable.